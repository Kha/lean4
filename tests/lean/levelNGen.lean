opaque test1 {α : Sort _} : α → Sort u_1
#check @test1
def test2 {α : Sort _} : α → Sort u_1 := sorry
#check @test2
