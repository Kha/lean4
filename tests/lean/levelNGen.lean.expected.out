@test1 : {α : Sort u_2} → α → Sort u_1
levelNGen.lean:3:4-3:9: warning: declaration uses 'sorry'
@test2 : {α : Sort u_2} → α → Sort u_1
